src/bctest.lisp

```diff
@@ -0,0 +1,378 @@
+;; hacks of lisp-unit tests from test/blockchain-test.lisp
```

Comment:
This does not refer to code presently considered part of the MVP.  Please remove.